tests/Tokenizer/TokensTest.php

```diff
@@ -741,6 +741,43 @@ public static function provideFindBlockEnd80Cases(): array
         ];
     }
 
+    /**
+     * @requires PHP 8.2
+     *
+     * @dataProvider provideFindBlockEnd82Cases
+     *
+     * @param Tokens::BLOCK_TYPE_* $type
+     */
+    public function testFindBlockEnd82(int $expectedIndex, string $source, int $type, int $searchIndex): void
+    {
+        static::assertFindBlockEnd($expectedIndex, $source, $type, $searchIndex);
+    }
+
+    public static function provideFindBlockEnd82Cases(): iterable
```

Comment:
Shouldn't there be test cases for DNF types in arrow functions and/or class' properties?

Similar in `DisjunctiveNormalFormTypeParenthesisTransformerTest`.

Reply:
If we test it in `DisjunctiveNormalFormTypeParenthesisTransformerTest` will adding the checks here bring any value?

Reply:
I don't know 🤷‍♂️😅. You know better probably, I think at least in one place would be enough.